legacy-tile-entities-v1/build.gradle

```diff
@@ -0,0 +1,7 @@
+archivesBaseName = "legacy-tile-entities-v1"
+version = getSubprojectVersion(project, "0.6.3")
+
+dependencies {
+	compile project(path: ':legacy-api-base')
+	compile project(path: ':legacy-registries-v0')
```

Comment:
from what I can tell, you only use registries module in test, right? make this `testImplementation`. Also will need to be registries-v1 bc it's now v1

Reply:
(the `Id` class is in api-base even tho it uses registry package)